applications/osb-portal/src/components/header/Header.tsx

```diff
@@ -5,14 +5,21 @@ import { Toolbar, Box, Button, Paper, Popper, MenuItem, MenuList, ClickAwayListe
 import makeStyles from '@mui/styles/makeStyles';
 import PersonIcon from "@mui/icons-material/Person";
 
+import {
+  headerBg,
+  secondaryColor
+} from "../../theme";
+
 const title = "Open Source Brain";
 
 const useStyles = makeStyles((theme) => ({
   toolbar: {
-    backgroundColor: theme.palette.background.paper,
+    backgroundColor: headerBg,
     paddingRight: theme.spacing(1),
     paddingLeft: theme.spacing(1),
     justifyContent: "space-between",
+    minHeight: '30px',
+    height: '30px'
```

Comment:
if it can be avoided, better to only set the min height, unless we need to override a default height